secantTests/RootTests/AppInitializationTests.swift

```diff
@@ -57,7 +57,9 @@ class AppInitializationTests: XCTestCase {
         store.dependencies.restoreWalletStorage = .noOp
 
         // Root of the test, the app finished the launch process and triggers the checks and initializations.
-        await store.send(.initialization(.appDelegate(.didFinishLaunching)))
+        await store.send(.initialization(.appDelegate(.didFinishLaunching))) { state in
+            state.appStartState = .didFinishLaunching
+        }
 
         await testQueue.advance(by: 0.02)
```

Comment:
Is this supposed to be the same as the delay that was increased to 0.5s?

Reply:
No, this has nothing to do with the time. There's an enum of all possible app starts and this is one of those cases, set of the type of the start.